README.md

```diff
@@ -82,5 +82,11 @@ end
 
 const DAG = [PersonGenerator(3:5) => [VisitGenerator(1:4) => [SymptomGenerator(1:2)]]]
 # pass RNG for reproducible generation:
-collect(MockTableGenerators.generate(StableRNG(11), DAG))
+results = collect(MockTableGenerators.generate(StableRNG(11), DAG))
+
+# Alternatively, linear DAGs can be also constructed in a flat representation:
```

Comment:
maybe with a little compat mention, so folks on earlier versions don't get confused:
```suggestion
# Alternatively, since v0.2.1, linear DAGs can be also constructed in a flat representation:
```